File: Dissidia.League.App/Dissidia.League.Domain/ValueObjects/Gamification/Pontuation/PlayerPontuation.cs
namespace Dissidia.League.Domain.ValueObjects.Gamification.Pontuation
{
    public class PlayerPontuation
    {
        public string Name { get; private set; }
        public int Wins { get; private set; }
        public int Losts { get; private set; }
        public double PercentualWins { get { return ((Wins * 100) / TotalMatches); } private set { } }
        public double PercentualLosts { get { return ((Losts * 100) / TotalMatches); } private set { } }
        public int TotalMatches { get { return Wins + Losts; } private set { } }

        public PlayerPontuation(string name, int wins, int losts)
        {
            Name = name;
            Wins = wins;
            Losts = losts;
        }

    }
}

File: frontend/src/client/app/ranking/model/player.pontuation.ts
export class PlayerPontuation {
    name:string;
    wins:number;
    losts:number;
    percentualWins;number;
    percentualLosts:number;
    totalMatches:number;
    type:number;

    constructor(name:string, type:number){
        this.name = name;
        this.wins = 0;
        this.losts = 0;
        this.percentualLosts = 0;
        this.percentualWins = 0;
        this.totalMatches = 0;
        this.type = type;
    }

}